src/pages/Comments/hooks/useInsertComment.ts

```diff
@@ -0,0 +1,44 @@
+import { gql } from "graphql-request";
+import { useMutation } from "react-query";
+import { InsertCommentMutation, InsertCommentMutationVariables } from "src/generated/graphql";
+
+import useQueryClient from "src/hooks/useQueryClient";
+
+const useInsertComment = (refetchComments: () => void) => {
+    const queryClient = useQueryClient();
+
+    const result = useMutation<InsertCommentMutation, Error, InsertCommentMutationVariables>(
+        ["InsertComment"],
+        async (variable) => {
+            const result = await queryClient.request<
+                InsertCommentMutation,
+                InsertCommentMutationVariables
+            >(
+                gql`
+                    mutation InsertComment(
+                        $authorId: uuid = ""

```

Comment:
I don't want blank string in my db

Reply:
Okay, I have just created a validate prop to handle validate to prevent empty string in CommentForm component.